### PR TITLE
fix(android/engine): Display welcome.htm help within the app

### LINKED ISSUE #4421: feat(android): Display welcome.htm with in-app WebView
> Is the welcome content being displayed in a web browser? It doesn't look like it is being displayed in-app.
> Okay, well that sounds like a definite candidate for improvement in a future update.

_Originally posted by @mcdurdin in https://github.com/keymanapp/keyman/issues/4413#issuecomment-772710428_

### PR DESCRIPTION
**Pull request by darcywong00**
Fixes #4421, follow-on to #4413 where displaying welcome.htm was separated from package installation.

This PR
* A new `KMHelpFileActivity` to display local welcome.htm within the app (online help.keyman.com keyboard help still goes to separate browser). "OK" button at the bottom prompts the user. This activity replaces HelpFile in:
    * KeyboardInfoActivity (KMEA)
    * ModelInfoActivity (KMEA)
    * KeyboardSettingsActivity (KMAPro)
    * MainActivity (KMAPro)
* Repurposes HelpFile for displaying accompanying pdf's via `Intent.ACTION_VIEW`

Note: two of the above activities are in KMEA, hence no Keyman tri-color bar styling in the titlebar.

Some things that got moved from KMAPro to KMEA
* welcome string
* titlebar.xml layout

Also threw in an update to the README to document pandoc as a prerequisite 

## Screenshots

sil_cameroon_qwerty welcome.htm
cameroon

khmer_angkor welcome.htm
khmer

khmer_angkor pdf link
pdf

### CODE REVIEW

File: android/KMEA/app/src/main/java/com/tavultesoft/kmea/util/HelpFile.java [hunk 1]
```diff
@@ -36,15 +40,16 @@
...
+        && ! KMManager.isTestMode()) {
       File customHelp = new File(new File(helpFile).getAbsolutePath());
       i.setFlags(Intent.FLAG_GRANT_READ_URI_PERMISSION);
       // Starting with Android N, you can't pass file:// to intents, so we use FileProvider
       try {
         final String authority = FileProviderUtils.getAuthority(context);
         Uri contentUri = FileProvider.getUriForFile(
           context, authority, customHelp);
-        i.setDataAndType(contentUri, "text/html");
+        i.setDataAndType(contentUri, "application/pdf");
```
Comment (mcdurdin) [inline android/KMEA/app/src/main/java/com/tavultesoft/kmea/util/HelpFile.java:52]: Will this set the type to `application/pdf` for `welcome.htm` as well as `.pdf` files?

**Review by mcdurdin — APPROVED** [review]
LGTM apart from that one question

File: android/KMEA/app/src/main/java/com/tavultesoft/kmea/util/HelpFile.java (hunk 1 above)
Comment (darcywong00) [reply android/KMEA/app/src/main/java/com/tavultesoft/kmea/util/HelpFile.java:52]: With the changes on this PR, html files are handled by KMHelpFileActivity, while only pdf files are handled by HelpFile.

I'll clean HelpFile to make this clear.

**Comment by keyman-server** [comment]
Changes in this pull request will be available for download in [Keyman version 14.0.243-beta](https://keyman.com/downloads/releases/beta/14.0.243)

**Comment by keyman-server** [comment]
Changes in this pull request will be available for download in [Keyman version 15.0.17-alpha](https://keyman.com/downloads/releases/alpha/15.0.17)

**Pull request merged**